pkg/systemd/generate/common.go

```diff
@@ -31,10 +31,12 @@ func validateRestartPolicy(restart string) error {
 }
 
 const headerTemplate = `# {{{{.ServiceName}}}}.service
```

Comment:
What do you think about making this part of containers.conf and allowing users to edit it?  Then we would not need to add all of the plumbing for a new option and users could just modify the default template however they wanted?

@vrothberg Thoughts?

Reply:
I would say it would be another feature - making this code useless.
It is architecture decision and you have to discuss this before I move further.
So @rhatdan , @vrothberg please let me know.

Reply:
I think supporting this kind of use case would be even more difficult. This would no longer allow us to do breaking changes to the template such as https://github.com/containers/podman/pull/9035.
I think adding a simple cli option is simple and intuitive for users.

Reply:
> What do you think about making this part of containers.conf and allowing users to edit it? Then we would not need to add all of the plumbing for a new option and users could just modify the default template however they wanted?

I think we always need a CLI counterpart for containers.conf options such that users can override defaults.  I have not strong feeling whether it's a good candidate for containers.conf.

But, I think it's something orthogonal to this change and we can add that a later point.

Reply:
@jmguzik feel free to continue working on it.   It would not change the architecture or logic of this PR.   If we want a counterpart in the containers.conf, we can add it separately.

Reply:
@vrothberg Ok, thanks for the clarification!